tests/unit/maze_transformer/training/test_tokenizer.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+
+from maze_transformer.generation.generators import LatticeMazeGenerators
+from maze_transformer.training.mazedataset import MazeDatasetConfig
+from maze_transformer.training.tokenizer import SPECIAL_TOKENS, MazeTokenizer
+
+
+def test_coordinate_system():
+    """
+    Check that the adjlist created by .as_tokens() uses the same coordinate system as the LatticeMaze adjlist.
+
+    To test this, generate both adjlists, sort them and convert to a common format, and check that they are equal.
+    """
+    maze_size = 3
+    maze = LatticeMazeGenerators.gen_dfs((maze_size, maze_size))
+    maze_adjlist = maze.as_adjlist()
+
+    # convert to the same format as the tokenizer adjlist
+    maze_adjlist_connections = [
+        [f"({c[0]},{c[1]})" for c in conn] for conn in maze_adjlist
+    ]
+
+    node_token_map = MazeDatasetConfig(
+        grid_n=maze_size, name="test", n_mazes=1
+    ).node_token_map
+
+    tokenized_maze = MazeTokenizer(
+        maze=maze,
+        solution=np.array(
+            maze.find_shortest_path(
+                c_start=(0, 0),
+                c_end=(maze_size - 1, maze_size - 1),
+            )
+        ),
```

Comment:
I don't think we need to compute the solution, we could just set the value to `np.array([[0,0]])`. Though maybe it's just cleaner to do it it as you have. Not sure! Thoughts?